<launch>
	<rosparam command="load" file="$(find nucleof746zg_IMU_driver)/config/mag_param.yaml"/>
	<node name="nucleof746zg_driver" pkg="nucleof746zg_IMU_driver" type="nucleof746zg_IMU_driver" output="screen"/>
	<node name="imu_filter_complementary" pkg="imu_complementary_filter" type="complementary_filter_node" >
		<remap from="/imu/data_raw" to="f746zg_imu/data_raw" />
		<remap from="/imu/mag" to="f746zg_imu/mag" />
		<param name="use_mag" value="true" />
		<param name="gain_mag" value="0.0005" />
		<param name="gain_acc" value="0.02" />
    </node>
</launch>
